RockWeb/Themes/NewSpring/assets/Lava/LAYOUT/editorial-detail.lava

```diff
@@ -44,12 +61,13 @@
     </style>
 {% endif %}
 
+
 {% comment %}
     Calculate prev/next entry links
 {% endcomment %}
 {% if itemsManuallyOrdered == 'true' %}
     {% assign currentSlug = 'Global' | PageParameter:'Slug' %}
-    {% assign itemIndex = parentItem.Children | Where:'Slug', currentSlug | Select:'Index' | First %}
+    {% assign itemIndex = parentItem.Children | Where:'Slug', currentSlug | Select:'Index' | First | Plus:1 %}
```

Comment:
Sorted out some issues with devotional indexes - initially we ran a sql command to set these as 1 based when Rock handles them as 0 based. Updated lava to expect 0 based.